sound/soc/sof/ipc4-topology.h

```diff
@@ -207,7 +207,7 @@ struct sof_ipc4_control_data {
 
 	union {
 		struct sof_ipc4_ctrl_value_chan chanv[0];
-		struct sof_abi_hdr data[0];
+		struct sof_ipc4_abi_hdr data[0];
```

Comment:
we probably need to clarify that this is an ABI change with userspace but since there are no users so far it's fine.

Reply:
Got it.